Amazon A2I Fraud Detector.ipynb

```diff
@@ -0,0 +1,796 @@
+    "plt.legend(loc=\"lower right\",fontsize=12)\n",
+    "plt.axvline(x = 0.02 ,linewidth=2, color='r')\n",
+    "plt.axhline(y = 0.73 ,linewidth=2, color='r')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Test the fraud detector with a sample data record\n",
+    "Using the fraud detector client, invoke the model endpoint with a sample record and examine the results including fraud detection score"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "eventId = uuid.uuid1()\n",
+    "timestampStr = '2013-07-16T19:00:00Z'\n",
+    "\n",
+    "# Construct a sample data record\n",
+    "\n",
+    "rec = {\n",
+    "   'ip_address': '36.72.99.64',\n",
+    "   'email_address': 'fake_bakermichael@example.net',\n",
+    "   'billing_state' : 'NJ',\n",
+    "   'user_agent' : 'Mozilla',\n",
+    "   'billing_postal' : '32067',\n",
+    "   'phone_number' :'703-989-7890',\n",
+    "   'user_agent' : 'Mozilla',\n",
+    "   'billing_address' :'12351 Amanda Knolls Fake St'\n",
+    "}\n",
+    "\n",
+    "\n",
+    "pred = client.get_event_prediction(detectorId=DETECTOR_NAME, \n",
+    "                                       detectorVersionId='1',\n",
+    "                                       eventId = str(eventId),\n",
+    "                                       eventTypeName = EVENT_TYPE,\n",
+    "                                       eventTimestamp = timestampStr, \n",
+    "                                       entities = [{'entityType': ENTITY_TYPE, 'entityId':str(eventId.int)}],\n",
+    "                                       eventVariables=rec) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "pred"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract/print the model score\n",
+    "pred['modelScores'][0]['scores']['sample_fraud_detection_insightscore']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Control Plane Resources\n",
+    "\n",
+    "Here we'll be constructing the following control plane resources: Human Task UI and Flow Definition, using the SageMaker CreateTaskUI and CreateFlowDefinition APIs, respectively.\n",
+    "\n",
+    "These resources can be created once and used to drive any subsequent A2I human loops.\n",
+    "\n",
+    "NOTE: The following template models a \"Claim\" - i.e. mark if a given claim is fraudulent, valid claim or needs further investigation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template=\"\"\"<script src=\"https://assets.crowd.aws/crowd-html-elements.js\"></script>\n",
```

Comment:
This should be moved to Create Human Task UI section

Reply:
moved